package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "portfolio_nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build && next export",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.12.3",
```

Comment:
It looks like a lot of these dependencies aren't even used. I would remove the unnecessary ones as it just increases your bundle size necessarily and can slow down page loads.